cpp/src/transform/jit/masked_udf_kernel.cu

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Include Jitify's cstddef header first
```

Comment:
Why? The convention in cudf is to include from "near" to "far". So, you include `<transform/...>` first, then `<cudf/...>`, then `<cuda/...>`, then std headers finally.

Reply:
I think the problem here is that technically when this file is runtime compilated later, `transform/jit/operation-udf.hpp` gets string replaced by by an actual function definition that might contain the types in the std headers. So I think at least the order of those two headers is critical.